src/Oscoin/Storage/Block/Pure.hs

```diff
@@ -100,6 +102,16 @@ getBlocks (fromIntegral -> d) Handle{..} =
 -- Nb. we guarantee that there is at least one chain in the store by exposing
 -- only the 'genesisHandle' smart constructor.
 
+-- | Gets a suffix of the 'BlockChain' originating from the input 'BlockHash'.
```

Comment:
```suggestion
-- | Gets a suffix of the 'Blockchain' originating from the input 'BlockHash'.
```
Not sure haddock is case-sensitive, but...

Reply:
a42f62e